python/mxnet/contrib/onnx/_import/op_translations/negative.py

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# coding: utf-8
+def negative(op_name, attrs, inputs):
```

Comment:
why create so many files? You can put all ops in one file

Reply:
As we implement more operators, it will become unwieldy to have all operator translations in the same file. The code will also be unclear. I was thinking maybe we could club similar operators into the same file like here - https://mxnet.apache.org/api/python/symbol/symbol.html#the-symbol-class

So all the arithmetic operators will be in one file, indexing routines like slice and gather in another file and so on. Will be a good balance between clubbing all the operators into the same file and having a different file for each operator.

Also some operator translations are much more involved than the ones I have made in this PR.

Reply:
@anirudhacharya Right now it seems like you are putting every single operator in a different file. (file names are also based on the operator defined).
Also, while implementing export you will have to do a reverse mapping between operators. I think that's just too many files to maintain.

+1 to have all operator conversions in a single file.